src/Util/Constants.ts

```diff
@@ -14,4 +14,6 @@ export const COMMAND_NAMES = {
   DEVELOPERS: 'developers',
   LOKI: 'loki',
   MEMBER_ACTIVITY: 'memberactivity'
-} as const;
\ No newline at end of file
+} as const;
+
+export type COMMAND_OPTION_TYPE = 'string';
```

Comment:
Hey, this line seems extraneous - any way we could just get rid of it?

Best,

C